spark-2.0-on-suse/azuredeploy.json

```diff
@@ -0,0 +1,419 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUserName": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The Linux VM user name"
+      }
+    },
+    "adminPassword": {
+	  "type": "securestring",
+      "metadata": {
+        "description": "The Linux VM password"
+      }
+    },
+    "numberOfSlaves" : {
+      "type": "int",
+      "metadata": {
+      "description": "Number Of Slaves"
+      },
+      "defaultValue": 1
```

Comment:
if we really need this for the sample is there a min & max we can add here then?